faq.rst

```diff
@@ -330,3 +332,18 @@ Or using the Web-GUI
 3. add "  revisionHistoryLimit: 2" between spec and replicas, on the same level as replicas
 
 More information about this in the OpenShift documentation: https://docs.openshift.com/container-platform/3.11/dev_guide/deployments/how_deployments_work.html#creating-a-deployment-configuration
+
+I get an error like 'You have reached your pull rate limit. You may increase the limit by authenticating and upgrading'
+-----------------------------------------------------------------------------------------------------------------------
+
+Docker Inc. has introduced a pull rate limited for container images stored on Docker Hub. If that limit is reached you get an error like:
+
+.. code-block:: console
+
+  toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
+
+To avoid this issue, you should always use authenticated image pulls for Docker Hub. You can do this by creating an account on https://hub.docker.com and adding a pull secret to your project, see :ref:`faq-pull-secret`.
+
+If you are using a free account on Docker Hub, this will increase your limit to 200 container image requests per 6 hours. If that is not enough and you are still reaching the limit you can upgrade your Docker Hub subscription to get unlimited container image pulls.
+
```

Comment:
I think we should add the info that a different registry as Quay.io, AWS ECR, GitLab Container Registry do currently not have this restriction for public container images.

Reply:
I added the information that migrating to other registries is another possibility to mitigate the issue.